asreview/webapp/src/StyledComponents/StyledAlert.js

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import { Alert } from "@mui/material";
+
+export function ExplorationModeRecordAlert(props) {
+  return (
+    <Alert
+      severity="info"
+      sx={{ borderBottomRightRadius: 0, borderBottomLeftRadius: 0 }}
+    >
+      This record was pre-labeled as relevant
```

Comment:
I'm not sure about the word "pre-labeled". Had this discussion also with @SagevdBrand before on her blog.

"This record was labeled relevant in an earlier study."

This can be the alternative.

Reply:
Agree. Changed.